Source/Widgets/Viewer/Viewer.js

```diff
@@ -1604,12 +1605,19 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
     }
 
     function updateTrackedEntity(viewer) {
-        if (!viewer._needTrackedEntityUpdate) {
+        var trackedEntity = viewer._trackedEntity;
+        var currentTime = viewer.clock.currentTime;
+
+        //Verify we have a current position at this time. This is only triggered if a position
+        //has become undefined after trackedEntity is set but before the boundingSphere has been
+        //computed. In this case, we will track the entity once it comes back into existence.
+        var currentPosition = Property.getValueOrUndefined(trackedEntity.position, currentTime);
```

Comment:
Wow, how did I miss that.  `Viewer` itself is hosed in master.  I'll fix it there directly since it's totally broken.